srcs/builtin_fts/env.c

```diff
@@ -22,10 +22,10 @@ int	env(t_cmd_table *cmd, t_minishell *minishell)
 	{
 		if (get_env_lst_value(curr))
 		{
-			ft_putstr_fd(get_env_lst_name(curr), cmd->fd_in);
-			write(cmd->fd_in, "=", 1);
-			ft_putstr_fd(get_env_lst_value(curr), cmd->fd_in);
-			write(cmd->fd_in, "\n", 1);
+			ft_putstr_fd(get_env_lst_name(curr), cmd->fd_out);
+			write(cmd->fd_out, "=", 1);
+			ft_putstr_fd(get_env_lst_value(curr), cmd->fd_out);
+			write(cmd->fd_out, "\n", 1);
```

Comment:
Why did you change everything to fd_out ? What was wrong ?

Reply:
That 😂 I was really stupid, I was writting to fd_in instead of fd_out 🤦‍♀️ and then of course it wasn't working

Reply:
hahahaha